app/models/generic_event/move_redirect.rb

```diff
@@ -24,13 +25,6 @@ def trigger
       eventable.move_type = move_type if move_type.present?
     end
 
-    def for_feed
-      super.tap do |common_feed_attributes|
-        common_feed_attributes['details'] = to_location.for_feed(prefix: 'to')
-        common_feed_attributes['details']['move_type'] = move_type if move_type.present?
```

Comment:
This attribute doesn't get replaced by the new shared module - was it removed intentionally?

Reply:
So this is a bit silly. Essentially the move type if it's set to anything will get set again in the details. If it's not set it will not get set. It's basically doing nothing of value and was clearly an early brain fart.

Reply:
Thanks for the explanation 👍